src/app/index.js

```diff
@@ -1,69 +1,208 @@
-var slideIndex, slides, dots;
-window.onload = function() {
+/**============= Require all the scss and js files =================== */
+import '../../style.scss';
+import { makeRequest } from '../app/core/services/sabkabazar.service';
+import { home } from '../app/home/home';
+import { products, onClickCategory } from '../app/products/products';
+import { login } from '../app/login/login';
+import { signup } from '../app/signup/signup';
```

Comment:
Instead of repeating the package name twice like ../app/**signup/signup** why don't you place an index.js file in ../app/signup and return the component/module from there?